### DISCUSSION TITLE
After installing the driver, it is not available

### DISCUSSION THREAD

**Discussion by liuzhichao03** [nomi-san/parsec-vdd#67] [Q&A]
After installing the driver, in the device manager, the device displays a yellow exclamation mark and error message code 31

**Comment by nomi-san** [comment]
Tell me your Windows version and the driver version did you use.
Did it work properly before this yellow exclamation? Or was it your first use?

**Reply by fish-candy** [reply]
I have the same problem.My Windows version is Windows 11 21H2 22000.318 . The driver version is v1.0.2.It was nervber working correctly.It's my first use.
![屏幕截图(2)](https://github.com/user-attachments/assets/9e102590-3a0c-49f4-94b8-9eabdd4c01b6)
![Uploading 屏幕截图(3).png…]()

**Reply by fish-candy** [reply]
it's another picture.One of the pictures in the previous comment was not uploaded well. 
![屏幕截图(3)](https://github.com/user-attachments/assets/c0f2f8a2-49cc-44d0-a0f3-39a2f2c074ce)

**Reply by nomi-san** [reply]
The issue is caused by your Windows or the driver itself.

Uninstall the driver and restart your PC, then reinstall it and restart your PC.
If not solved, try selecting parent GPU in this [guide](https://support.parsec.app/hc/en-us/articles/32361359271444-VDD-Advanced-Configuration) and restart your PC.

**Comment by faryyy805** [comment]
I also have this problem，and it doesn't work after “selecting parent GPU in this [guide](https://support.parsec.app/hc/en-us/articles/32361359271444-VDD-Advanced-Configuration)”
![Snipaste_2025-04-15_15-53-31](https://github.com/user-attachments/assets/63676815-9054-4106-9a6f-416445eb0949)

**Reply by faryyy805** [reply]
![image](https://github.com/user-attachments/assets/3b99ea35-36a7-4635-9f14-1cf885f15d92)
![image](https://github.com/user-attachments/assets/46b62ca7-3cff-4677-83b6-3c2c2675c617)

**Reply by nomi-san** [reply]
Close the app and reinstall the driver (clean uninstall with safe mode if possible), then run Parsec app to check the driver state.
